low-level/sspl-ll.spec

```diff
@@ -105,19 +102,21 @@ chmod 644 $STATE_FILE
 %post
 mkdir -p /var/%{product_family}/sspl/bundle /var/log/%{product_family}/sspl /etc/sspl
 SSPL_DIR=/opt/seagate/%{product_family}/sspl
+SSPL_SETUP=$SSPL_DIR/low-level/files/opt/seagate/sspl/setup
+cp -p $SSPL_SETUP/generate_resource_health_view/resource_health_view $SSPL_SETUP/
+cp -p $SSPL_SETUP/generate_sspl_bundle/sspl_bundle_generate $SSPL_SETUP/
 
 [ -d "${SSPL_DIR}" ] && {
-    ln -sf $SSPL_DIR/low-level/framework/sspl_ll_d /usr/bin/sspl_ll_d
-    ln -sf $SSPL_DIR/low-level/framework/sspl_ll_d $SSPL_DIR/bin/sspl_ll_d
-    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/generate_resource_health_view/resource_health_view /usr/bin/resource_health_view
-    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate /usr/bin/sspl_bundle_generate
-    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate $SSPL_DIR/bin/sspl_bundle_generate
-    ln -sf $SSPL_DIR/low-level/files/opt/seagate/sspl/setup/manifest_support_bundle /usr/bin/manifest_support_bundle
-    ln -sf $SSPL_DIR/low-level/framework/sspl_rabbitmq_reinit.py $SSPL_DIR/bin/sspl_rabbitmq_reinit
-    ln -sf $SSPL_DIR/low-level/framework/base/sspl_constants.py $SSPL_DIR/bin/sspl_constants.py
-    ln -sf $SSPL_DIR/low-level/framework/sspl_init $SSPL_DIR/bin/sspl_init
-    ln -sf $SSPL_DIR/low-level/framework/sspl_reinit $SSPL_DIR/bin/sspl_reinit
-    ln -sf $SSPL_DIR/low-level/framework/utils/salt_util.py $SSPL_DIR/bin/salt_util.py
+    ln -sf $SSPL_SETUP/resource_health_view $SSPL_DIR/bin/resource_health_view
+    ln -sf $SSPL_SETUP/sspl_bundle_generate $SSPL_DIR/bin/sspl_bundle_generate
+    ln -sf $SSPL_SETUP/manifest_support_bundle $SSPL_DIR/bin/manifest_support_bundle
+    ln -sf $SSPL_SETUP/sspl_setup.py $SSPL_DIR/bin/sspl_setup
+    ln -sf $SSPL_SETUP/sspl_dev_deploy $SSPL_DIR/bin/sspl_dev_deploy
+    ln -sf $SSPL_SETUP/pacemaker $SSPL_DIR/bin/pacemaker
+    ln -sf $SSPL_SETUP/consuldump.py $SSPL_DIR/bin/consuldump.py
+    ln -sf $SSPL_SETUP/resource_health_view /usr/bin/resource_health_view
```

Comment:
i believe following scripts were asked in /usr/bin during design & deveopment in CortxV1 timeframe by driving components. In which case, its fine
/usr/bin/resource_health_view
/usr/bin/sspl_bundle_generate
/usr/bin/manifest_support_bundle

Reply:
Done